### PR TITLE
Modify Japanese translation

### PR DESCRIPTION
**Pull request by nakaokat**
The original text is "when the format is unambiguous". The Japanese translation meant "when the format is ambiguous", so I modified it.

### CODE REVIEW

**Comment by eapache** [comment]
Thanks! While checking this in Google Translate I also noticed that the original text uses "and" whereas this still translates with an "or". I'm not sure if that's Google's mistake or if that is also worth correcting here?

**Comment by nakaokat** [comment]
I think it's trivial, but I try to correct that.

File: lang/TUTORIAL_JAPANESE.md [hunk 1]
```diff
@@ -719,7 +719,7 @@ GraphQLの草案は素朴なものよりもかなり良くなっていますが
 処理を簡単にするために、クライアント側で事前に検証を行うことが難しい場合には、我々は入力に対して意図的に弱い型付けを行います。
 これにより、ビジネスロジックレイヤーがすべての検証を行い、クライアントも一箇所から発生するエラーだけに対処するだけで済みます。
 
-*ルール #19: フォーマットが曖昧であったり、クライアント側の検証が複雑な場合には、入力に対して弱い型付け（`Email`ではなく`String`）を行うこと。これによりサーバーは一度にすべての検証を実行し、単一のフォーマットでエラーを報告することになり、結果としてクライアントが非常にシンプルになる。*
+*ルール #19: フォーマットが明確であったり、クライアント側の検証が複雑な場合には、入力に対して弱い型付け（`Email`ではなく`String`）を行うこと。これによりサーバーは一度にすべての検証を実行し、単一のフォーマットでエラーを報告することになり、結果としてクライアントが非常にシンプルになる。*
```
Comment (nakaokat) [inline lang/TUTORIAL_JAPANESE.md:722]:
```suggestion
*ルール #19: フォーマットが明確であり、クライアント側の検証が複雑な場合には、入力に対して弱い型付け（`Email`ではなく`String`）を行うこと。これによりサーバーは一度にすべての検証を実行し、単一のフォーマットでエラーを報告することになり、結果としてクライアントが非常にシンプルになる。*
```

File: lang/TUTORIAL_JAPANESE.md [hunk 2]
```diff
@@ -842,7 +842,7 @@ type CollectionUpdatePayload {
 - ルール #16: 関連に対するmutationを分けて実装するときには、一度に複数の要素に対して実行することが便利かどうか検討すること。
 - ルール #17: アルファベット順でグループ化するために、mutationの接頭辞に操作対象のオブジェクト名を用いること（例えば`cancelOrder`ではなく`orderCancel`とする。）
 - ルール #18: mutation処理を遂行するうえで意味的に本当に必要である場合に限り、入力フィールドを必須とすること。
-- ルール #19: フォーマットが曖昧であったり、クライアント側の検証が複雑な場合には、入力に対して弱い型付け（`Email`ではなく`String`）を行うこと。これによりサーバーは一度にすべての検証を実行し、単一のフォーマットでエラーを報告することになり、結果としてクライアントが非常にシンプルになる。
+- ルール #19: フォーマットが明確であったり、クライアント側の検証が複雑な場合には、入力に対して弱い型付け（`Email`ではなく`String`）を行うこと。これによりサーバーは一度にすべての検証を実行し、単一のフォーマットでエラーを報告することになり、結果としてクライアントが非常にシンプルになる。
```
Comment (nakaokat) [inline lang/TUTORIAL_JAPANESE.md:845]:
```suggestion
- ルール #19: フォーマットが明確であり、クライアント側の検証が複雑な場合には、入力に対して弱い型付け（`Email`ではなく`String`）を行うこと。これによりサーバーは一度にすべての検証を実行し、単一のフォーマットでエラーを報告することになり、結果としてクライアントが非常にシンプルになる。
```

**Comment by eapache** [comment]
Thank you!

**Pull request merged**